invenio_communities/communities/records/systemfields/community_theme.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+"""Community theme systemfield."""
+
+from invenio_records.systemfields import SystemField
+
+
+class CommunityThemeField(SystemField):
+    """Systemfield for controlling community theme indexing."""
+
+    #
+    # Data descriptor methods (i.e. attribute access)
+    #
+    def __get__(self, record, owner=None):
+        """Get the deletion status of the community."""
+        if record is None:
+            return self  # returns the field itself.
+
+        return record.get("theme")
+
+    def pre_dump(self, record, data, **kwargs):
+        """Remove theme information from indexing."""
+        record.pop("theme", None)
```

Comment:
This systemfield is used just to pop the information before indexing. Maybe it is an overkill? Do we prefer a search dumper?